src/__stories__/logo-story.tsx

```diff
@@ -78,6 +88,7 @@ Default.args = {
     inverse: false,
     forceBrandLogo: false,
     brand: 'Movistar',
+    color: '',
```

Comment:
instead of a empty box, I prefer to give a list of examples like in progress bar story
https://mistica-ekone7k5g-flows-projects-65bb050e.vercel.app/?path=/story/components-progress-bars--progress-bar-story

For example:
`color: 'default' | 'neutralHigh' | 'neutralMedium' | '#000000';`

Reply:
ok, done